gen-cpp/CppRacconChat.h

```diff
@@ -0,0 +1,18 @@
+#include "RacconChat.h"
```

Comment:
the same error
![image](https://cloud.githubusercontent.com/assets/13332523/21955423/00402bb4-da7c-11e6-950f-85ad3ef46b20.png)